resources/views/store/manage_vouchers.blade.php

```diff
@@ -164,7 +164,7 @@
                         </div>
                     </div>
                 </div>
-                <button class="short-button">Confirm pick up</button>
+                <button class="short-button" onclick="javascript:window.location.href='{{ URL::route("store.registration.index") }}'; return false;">Confirm pick up</button>
```

Comment:
Since this is basically "yep, lets pretend we did that" at the moment, rather than actually doing a collection, this is OK

Reply:
yup!